refined/src/test/scala/magnolify/refined/test/RefinedTestSuite.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package magnolify.refined.test
+
+import java.nio.ByteBuffer
+
+import magnolify.avro.test._
+import magnolify.bigquery.test._
+import magnolify.bigtable.test._
+import cats.implicits._
+import eu.timepit.refined.scalacheck.string._
+import eu.timepit.refined.scalacheck.numeric._
+import eu.timepit.refined.scalacheck.char._
+import eu.timepit.refined.char.{Digit, UpperCase}
+import magnolify.refined._
+import magnolify.cats.auto._
+import magnolify.scalacheck.auto._
+import magnolify.refined.test.TestUtils.WordCount
+
+object TestUtils {
+  import eu.timepit.refined._
+  import eu.timepit.refined.api._
+  import eu.timepit.refined.boolean._
+  import eu.timepit.refined.numeric._
+  import eu.timepit.refined.string._
+
+  type LessThanHundred = Positive And Not[Greater[W.`100`.T]]
+  type Age = Int Refined LessThanHundred
+  type StartWithS = String Refined StartsWith[W.`"S"`.T]
+
+  type Count = Long Refined Positive
+  type Word = String Refined Trimmed
+
+  case class Person(name: StartWithS, age: Age)
+  case class WordCount(word: Word, count: Count)
+
+  case class BigtableRefExample(
+    c: Char Refined UpperCase,
+    i: Int Refined Positive,
+    o: Option[Char Refined Digit]
+  )
+}
+
+class RefinedAvroSuite extends AvroBaseSuite {
```

Comment:
These 3 suites don't test refined specific logic right? Just that existing `{Avro,TableRow,Bigtable}Type` work with `Refined`? Also `EntityType`, `ExampleType` & `ProtobufType` are missing. Do they not work or just not implemented?